custom/workflow-helper/app-deployer/README.md

```diff
@@ -0,0 +1,98 @@
+## About App-Deployer
+App-Deployer has been used for the installation of applications.
+
+At first, the user is asked to give the namespace, typeName, operation, app, scope, and timeout.
+
+<table>
+  <tr>
+    <th> Environment </th>
+    <th> Description </th>
+    <th> Example </th>
+  </tr>
+  <tr>
+    <td> -namespace=<"app-namespace"> </td>
+    <td> Application namespace has to pass </td>
+    <td> -namespace="sock-shop" </td>
+  </tr>
+  <tr>
+    <td> -typeName=<"scenario"> </td>
+    <td> It can be weak/resilient application </br>
+     Note: </br>
+     1. In a weak scenario, it will create a single replica and Deployments for all. </br> 
+     2. In a resilient scenario, it will create two replicas of pods with Statefulsets for databases and Deployments for others. </td>
+     <td> -typeName="resilient"</td>
+  </tr>
+  <tr>
+    <td> -timeout=<"timeout"> </td>
+    <td> Timeout is used for the termination of application </td>
+    <td> -timeout="400" </td>
+  </tr>
+  <tr>
+    <td> -operation=<"operation"> </td>
+    <td> Operation will be Kubernetes CRUD operations 
+         on these resources.
+    </td>
+    <td> -operation="apply" </td>
+  </tr>
+  </tr>
+    <td> -app=<"app-name"> </td>
+    <td> App is the name of the application </td>
+    <td> -app="sock-shop" </td>
+  </tr>
+    <td> -scope=<"scope"> </td>
+    <td> Scope of application </td>
+    <td> -scope="cluster" </td>
+  </tr>
+</table>
```

Comment:
```suggestion
<table>
  <tr>
    <th> Flag </th>
    <th> Description </th>
    <th> Example </th>
  </tr>
  <tr>
    <td> -namespace </td>
    <td> Namespace of the application </td>
    <td> -namespace="sock-shop" </td>
  </tr>
  <tr>
    <td> -typeName </td>
    <td> Type of application </br>
     Note: </br>
     1. In a weak scenario, it will create a single replica and Deployments for all</br> 
     2. In a resilient scenario, it will create two replicas of Statefulsets for databases and two replicas of Deployments for others.</td>
     <td> -typeName="resilient"</td>
  </tr>
  <tr>
    <td> -timeout </td>
    <td> Time after which it will fail the app-deployer if application is not ready </td>
    <td> -timeout="400" </td>
  </tr>
  <tr>
    <td> -operation</td>
    <td> Operation will be Kubernetes CRUD operations 
         on these resources. It supports create,delete, and apply operations.
    </td>
    <td> -operation="apply" </td>
  </tr>
  </tr>
    <td> -app </td>
    <td> Name of the application </td>
    <td> -app="sock-shop" </td>
  </tr>
    <td> -scope </td>
    <td> Scope of application. It supports cluster and namespace modes </td>
    <td> -scope="cluster" </td>
  </tr>
</table>
```